components/tensorboard-controller/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+  - elikatsis
+  - kandrio98
+  - kimwnasptd
+  - quanjielin
```

Comment:
I don't believe @quanjielin is working on this anymore.

Reply:
Do you propose to not include him in approvers, reviewers or both?

Reply:
Both; I don't think she is working on KF anymore; 

@quanjielin are you still working on TB controller or can we remove you from the OWNERs file?

Reply:
feel free to remove me from the owner files.

Reply:
ACK, I removed her from the OWNERs file